cockroachcloud/free-faqs.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Frequently Asked Questions
+summary: Get answers to frequently asked questions about CockroachCloud Free (beta)
+toc: true
+redirect_from:
+- ../stable/cockroachcloud-frequently-asked-questions.html
+---
+
+This page answers the frequently asked questions about CockroachCloud Free (beta). For answers to frequently asked questions about the paid version of CockroachCloud, see [CockroachCloud FAQs](frequently-asked-questions.html).
+
+## General
+
+### What is CockroachCloud Free?
+
+CockroachCloud Free delivers free CockroachDB clusters for you and your organization. It is a managed instance of CockroachDB that also removes the friction of initial cluster sizing and auto-scales based on your application traffic. While we have eliminated the concept of nodes for you, there will be an upper limit of usage of up to 1 vCPU and 5GB storage per free cluster.
+
+### What can I use CockroachCloud Free for?
+
+CockroachCloud Free can be used for proofs-of-concept, toy programs, or to use while completing [Cockroach University](https://www.cockroachlabs.com/cockroach-university/).
+
+For examples of applications that use CockroachCloud Free, check out the following [Hack the North](https://hackthenorth.com/) projects:
+
+- [flock](https://devpost.com/software/flock-figure-out-what-film-to-watch-with-friends)
+- [mntr.tech](https://devpost.com/software/mntr-tech)
+- [curbshop.online](https://devpost.com/software/curbshop-online)
+
+### What are the limitations of CockroachCloud Free?
+
+CockroachCloud Free is currently in beta and as such, it is not suitable for production use. Specifically, there are capabilities we are still working on enabling, such as the ability to enable backups, to import data, and no-downtime upgrades to a paid tier. These capabilities and more are coming soon!
```

Comment:
Can we say "not recommended for serious production use" 
Because toy apps/ hobby apps can be prod, I don't want us to necessarily place that value judgement.